src/estimation/bootstrap.jl

```diff
@@ -0,0 +1,59 @@
+# Should use ensemble problem
+struct Bootstraps
+  populations
+  fits
+  stratify_by
+end
+bootstrap(fpm::FittedPumasModel; kwargs...) = bootstrap(fpm.model, fpm.data, coef(fpm), fpm.approx; kwargs...)
+
+function try_fit(args...)
+  try
+    return fit(args...)
+  catch e
+    return e
+  end
+end
+
+function bootstrap(model::PumasModel, data::Population, coef, method; samples=200, stratify_by=nothing)
```

Comment:
```suggestion
function bootstrap(model::PumasModel, data::Population, coef::NamedTuple, approx::LikelihoodApproximation; samples=200, stratify_by=nothing)
```
I think we use `approx` everywhere else.